.github/workflows/test.yml

```diff
@@ -53,35 +53,9 @@ jobs:
   #       run: cd cosmovisor; make
   #       if: env.GIT_DIFF
 
-  split-test-files:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Create a file with all the pkgs
-        run: go list ./... > pkgs.txt
-      - name: Split pkgs into 4 files
-        run: split -d -n l/4 --elide-empty-files pkgs.txt pkgs.txt.part.
-      # cache multiple
-      - uses: actions/upload-artifact@v3
-        with:
-          name: "${{ github.sha }}-00"
-          path: ./pkgs.txt.part.00
-      - uses: actions/upload-artifact@v3
-        with:
-          name: "${{ github.sha }}-01"
-          path: ./pkgs.txt.part.01
-      - uses: actions/upload-artifact@v3
-        with:
-          name: "${{ github.sha }}-02"
-          path: ./pkgs.txt.part.02
-      - uses: actions/upload-artifact@v3
-        with:
-          name: "${{ github.sha }}-03"
-          path: ./pkgs.txt.part.03
```

Comment:
@Chirag018 since we're removing the split test files, we should also remove the `matrix` definitions below because now the jobs that needed `split-test-files` as a prerequisite don't need to be run across separate files